### DISCUSSION TITLE
Product price change doesn't change (sub)total price in cart?

### DISCUSSION THREAD

**Discussion by Kealtie** [swellstores/community#214] [Help] [answered]
I think this is a bug, but in the dashboard I changed the price of a certain product.

All the active carts containing this product still had a (sub)total price using the old price of the product.

**Comment by SusannaSwell** [comment, marked as answer]
hello @jordytenden this is the intended behaviour for now, however there is a feature improvement that's logged in order to update the cart in this case.

**Reply by Kealtie** [reply]
Hi @SusannaSwell, what do you mean with feature improvement? How could I update the cart to contain the new prices?

**Reply by ericingram** [reply]
@jordytenden you can update the price on any cart using the backend API, or by removing and re-adding a product.

The platform doesn't automatically update cart prices, but we would consider adding a setting to enable that behavior in the future.
